### PR TITLE
Add browse for endpoints, group in a 'network' item.

### LINKED ISSUE #407: Add Endpoints to the tree view
Hello, I just installed the kubernetes extension and I noticed that I am not able to see the Endpoints Resources. I know I can get the same data in the Pod view, but it would be nice to see the Endpoint Resource of a service and get a collection of IP addresses. Maybe it exists and I am not seeing it? But  I still have to use `kubectl get endpoints` in order to get this view.

Thanks!

### PR DESCRIPTION
**Pull request by brendandburns**
Closes #407

@itowlson @squillace please take a look as it changes the tree-view UX slightly.

### CODE REVIEW

File: src/kuberesources.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ export class ResourceKind implements vscode.QuickPickItem {
 }
 
 export const allKinds = {
+    endpoint: new ResourceKind("Endpoiont", "Endpoints", "Endpoint", "endpoints"),
```
Comment (bnookala) [inline src/kuberesources.ts]: "Endpoint" is misspelled - first parameter to `ResourceKind`

Comment (brendandburns) [reply src/kuberesources.ts]: fixed, thanks!

**Comment by brendandburns** [comment]
Comment addressed, please re-check.

**Comment by bnookala** [comment]
merging - https://github.com/Azure/vscode-kubernetes-tools/pull/404 should fix the build issue.

**Pull request merged**